src/Model/FileTag.php

```diff
@@ -290,7 +290,11 @@ public static function unsaveFile($uid, $item_id, $file, $cat = false)
             return false;
         }
 
-        $fields = ['file' => str_replace($pattern, null, $item['file'])];
+        $fields = ['file' => str_replace($pattern, '', $item['file'])];
+        if ($fields === '') {
+        	$fields = null;
```

Comment:
`$fields` is supposed to be an array of database table fields with their new value, so it can never be strictly equal to `''`.

Reply:
Ohh right! 
I wanted it to be `$fields = ['file' => null];`
Thanks!